docs/user/openstack/README.md

```diff
@@ -18,12 +18,17 @@ In addition, it covers the installation with the default CNI (OpenShiftSDN), as
     - [Nova Metadata Service](#nova-metadata-service)
     - [Glance Service](#glance-service)
   - [OpenStack Credentials](#openstack-credentials)
+    - [Self Signed OpenStack CA certificates](#self-signed-openstack-ca-certificates)
   - [Standalone Single-Node Development Environment](#standalone-single-node-development-environment)
   - [Running The Installer](#running-the-installer)
     - [Known Issues](#known-issues)
     - [Initial Setup](#initial-setup)
     - [API Access](#api-access)
       - [Using Floating IPs](#using-floating-ips)
+        - [Create API and Ingress Floating IP Addresses](#create-api-and-ingress-floating-ip-addresses)
```

Comment:
Isn't the current depth of the TOC enough? I'd like to avoid it turn into a huge wall of text.

Reply:
Ah the auto linter snuck that in. I'll take that out